README.md

```diff
@@ -33,7 +33,7 @@ docker run -it --rm --name zeppelin --net=host -e PORT0=[ZEPPELIN_PORT] \
   -v $(pwd)/conf/hadoop/:/etc/hadoop/conf/ \
   -v $(pwd)/conf/spark/spark-env.sh:/usr/local/spark/conf/spark-env.sh \
   -v $(pwd)/conf/spark/spark-defaults.conf:/tmp/spark-defaults.conf \
-  saagie/zeppelin:sp1.12.1-mesos1.3.1-spark2.1.0 /zeppelin/saagie-zeppelin.sh -d DEBUG --port [ZEPPELIN_PORT]
+  saagie/zeppelin:sp1.12.1-mesos1.3.1-spark2.3.4 /zeppelin/saagie-zeppelin.sh -d DEBUG --port [ZEPPELIN_PORT]
```

Comment:
SPARK_VERSION is 2.4.5 by default, maybe your example could use 2.4.5 as well?

Reply:
OK ! This was changed in December when upgrading 2.3.4 but I forgot to change it for this other upgrading.